src/app/components/recipes-and-filters/recipes-and-filters.component.html

```diff
@@ -6,7 +6,7 @@
     <div class="main-content-recipes maxed-width">
       <h1>List of Recipes</h1>
       <app-recipes-cmp *ngIf="recipes|async; else spinloading" [recipes]="recipes|async"></app-recipes-cmp>
-      <button aria-label="Toggle Filters" class="toggle-filters" md-fab (click)="filtersSidenav.toggle()"><md-icon [svgIcon]="'settings_filter'"></md-icon></button>
+      <button aria-label="Toggle Filters" class="toggle-filters mat-primary" (click)="filtersSidenav.toggle()"><md-icon [svgIcon]="'settings_filter'"></md-icon></button>
```

Comment:
yes this doesnt sound right. the mat-primary class should be added by material directly so it shouldnt be there. also its removing the md-fab which is the directive that turns into the "fab" button.